modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/service/internal/search/index/test/InstanceWorkflowMetricsIndexerTest.java

```diff
@@ -115,8 +128,86 @@ public void testReindex() throws Exception {
 			workflowDefinition.getWorkflowDefinitionId());
 	}
 
+	@Test
+	public void testUpdateAssigneeUser() throws Exception {
```

Comment:
@MarinhoFeliphe What do you think to break this in two tests? And do not user `AssigneeUser` in the method name. I think that just `blaBlaBlaAssignee` is enough.

Thx